tests/HasMediaConversionsTrait/DeleteMediaTest.php

```diff
@@ -39,6 +39,21 @@ public function it_can_clear_a_collection()
     }
 
     /** @test */
+    public function it_can_clear_a_collection_excluding_some_media()
```

Comment:
Test feels a bit brittle. It only asserts that the number of items is correct. It would be better to check if the ids of excluded items are still present after using `clearMediaCollectionExcept`